### PR TITLE
Update NOW API Docs Mar 20

### PR DESCRIPTION
**Pull request by dkluever22**
* Announces NOW API changes linked to release version 7.93
* Fixes errors in earlier change log entries
* Adds missing `orderUnitInfo` attribute path to PATCH /{shopId}/products/{productId} (clarified with Micha)
* Corrects descriptions of `energyLabelSourceFile` and `productDataSheet` as per ticket: https://epages.atlassian.net/browse/UC-4143
* Fixes issues in POST /{shopId}/coupon-campaigns as per ticket: https://epages.atlassian.net/browse/AD-11085

### CODE REVIEW

File: _raml/apps/coupon-campaign/coupon-campaigns.raml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #%RAML 0.8
 displayName: Coupon campaigns
 post:
-  description: Creates a coupon campaign for a shop. Coupon campaign types can be FIX (absolute value), `PERCENT` (percentage value), and `SHIPPING` (waive delivery costs). *epages6 *epagesNow
+  description: Creates a coupon campaign for a shop. Coupon campaign types can be `FIX` (absolute value), `PERCENT` (percentage value), and `SHIPPING` (waive delivery costs). *epages6 *epagesNow
   securedBy: [coupons_write]
-  is: [ locale ]
+  is: [ localeMandatory ]
```
Comment (dkluever22) [inline _raml/apps/coupon-campaign/coupon-campaigns.raml:6]: related to https://epages.atlassian.net/browse/AD-11085

File: _raml/apps/coupon-campaign/examples/post_coupon_campaign_request.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
   "minimumOrderValue": 100,
   "type": {
     "name": "FIX",
-    "waiveShippingCosts": false
```
Comment (dkluever22) [inline _raml/apps/coupon-campaign/examples/post_coupon_campaign_request.json:8]: related to https://epages.atlassian.net/browse/AD-11085

File: _raml/apps/coupon-campaign/examples/post_coupon_campaign_request.json [hunk 3]
```diff
@@ -5,6 +5,7 @@
...
+    "waiveShippingCosts": true,
```
Comment (dkluever22) [inline _raml/apps/coupon-campaign/examples/post_coupon_campaign_request.json:8]: related to https://epages.atlassian.net/browse/AD-11085

File: _raml/apps/coupon-campaign/examples/post_coupon_campaign_response.json [hunk 4]
```diff
@@ -5,7 +5,8 @@
   "currency": "EUR",
   "type": {
     "name": "FIX",
-    "waiveShippingCosts": false
+    "waiveShippingCosts": true,
```
Comment (dkluever22) [inline _raml/apps/coupon-campaign/examples/post_coupon_campaign_response.json:8]: related to https://epages.atlassian.net/browse/AD-11085

File: apps/data-types.md [hunk 5]
```diff
@@ -606,8 +606,8 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | availability | string | The availability of the product. Can be one of *OnStock*, *WarnStock*, *OutStock*. |
 | energyLabelsMaxEfficiencyString | string | The highest possible efficiency class of an uploaded energy label. |
 | energyLabelsString | string | A list of energy labels applied to this product. Can be one value or a range with two values. |
-| energyLabelSourceFile | string | A URL with an image or PDF file containing the energy label image supplied by the manufacturer. |
-| productDataSheet | string | An image or PDF file containing a data sheet with technical information on the product. Has to be available if the product has an energy label. |
+| energyLabelSourceFile | string | A URL with an image or PDF file containing the energy label image supplied by the manufacturer. Has to be available if the product has an energy label. |
```
Comment (dkluever22) [inline apps/data-types.md:609]: related to https://epages.atlassian.net/browse/UC-4143

**Review by nsimu — APPROVED** [review]

**Review by mhoehn — APPROVED** [review]

**Pull request merged**